ICAFIX/scripts/computeRecleanFeatures.m

```diff
@@ -7,40 +7,45 @@ function computeRecleanFeatures(StudyFolder, ...
                                 CorticalParcellationFile, ...
                                 SubRegionParcellationFile, ...
                                 WMLabelFile, ...
-                                CSFLabelFile)
+                                CSFLabelFile, VisualAreasFile, LanguageAreasFile, SubRegionsFile, NonGreyParcelsFile)
 
 HCPPIPEDIR = getenv('HCPPIPEDIR')
+FullGreyordinateLength = 91282; 
```

Comment:
This still counts as hardcoded, please use the input files to determine this instead.

Reply:
Hi Tim, I think none of these input arguments have the full 91282 greyordinate space...The cortical parcellation has 59412...

Reply:
There is a version on BALSA that is 91282, or you can make one with -cifti-create-dense-from-template in advance (or, like my other comments, do more of this dlabel-massaging work in the bash script before it calls matlab).

You make it 91282 specifically to match other cifti files that the matlab code uses, so you can check those (otherwise, you could pass in the relevant global/templates file).  But, even doing it that way, as my other comment says, assumes that the nonmatching stuff is always at the end (the cifti format does not require any specific order of structures, that is merely convention), so -cifti-create-dense-from-template (or a function using `cifti_struct_dense_extract_surface_data`, etc) is more robust and doesn't require hardcoding any internals of the standard grayordinates (numbers or ordering).